application/listeners/posthelpers.go

```diff
@@ -2,12 +2,29 @@ package listeners
 
 import (
 	"errors"
-
+	"fmt"
 	"github.com/silinternational/wecarry-api/domain"
 	"github.com/silinternational/wecarry-api/models"
 	"github.com/silinternational/wecarry-api/notifications"
 )
 
+// TODO create a proper solution for this
+func getUserLanguage() string {
+	return "en"
+}
+
+type argAppName struct{ AppName string }
+
+func getTranslatedSubject(translationID, template string) string {
+	subj, err := domain.TranslateWithLang(getUserLanguage(), translationID, argAppName{AppName: domain.Env.AppName})
```

Comment:
Would it make sense to include the `AppName` arg in the `TranslateWithLang` function? Might be an arg that is needed more often than not.

Reply:
Hopefully the `TranslateWithLang` function will be getting replaced by a near-future upgrade.